### PR TITLE
feat(team): Add new team members Rebecca Fordon and Michelle Ma

### LINKED ISSUE #218: Add Rebecca and Michelle to free.law/team
Rebecca Fordon's bio:

> Rebecca brings a unique perspective on legal technology and legal research, having transitioned into law librarianship after a successful career as a corporate bankruptcy attorney at an AmLaw 200 firm. A long-time volunteer with Free Law Project, her research interests have been greatly influenced by the organization’s mission, focusing currently on electronic court records, legal citators, and expanding open access to legal information. Rebecca frequently writes and speaks about legal technology and generative AI in the legal field and is a 2024 vLex Fastcase 50 honoree. She currently serves as a Law Librarian and Assistant Professor of Professional Practice at The Ohio State University’s Moritz College of Law, where she brings her expertise to both the classroom and the library. Rebecca holds degrees from Ohio Wesleyan University, Boston University School of Law, and UCLA and in her spare time loves crosswords, cooking, and cycling.

Her photo

And Michelle Ma's bio:

> Michelle Ma is an attorney and startup founder who is passionate about building legal technology for self-advocacy. She brings a decade of legal experience counseling technology companies on contracts, technology licensing, product and IP issues, and M&A. A former mechanical engineer, she worked as an executive and founder of several early stage startups before opening her own practice. 

Her Photo

----

We'll need to convert and downsize the photos, etc, but should be straightforward. Please let me know what questions arise!

### PR DESCRIPTION
**Pull request by elisa-a-v**
Resolves #218

### CODE REVIEW

**Comment by mlissner** [comment]
Looks great. I noticed the smallest screens are a bit funky. 

image

Looks like an existing problem, but worth fixing.

File: styles/global.css [hunk 1]
```diff
@@ -62,4 +62,8 @@
 }
 .prose code::before {
   content: "";
+}
+
+.team-container img {
+  margin-bottom: 0;
 }
```
Comment (mlissner) [inline styles/global.css:69]: You should be able to do this with `mb-0`, no? In general with tailwind, you almost never have to add new CSS.

**Review by mlissner — CHANGES_REQUESTED** [review]
This looks good, but needs one tweak. I think next time we work on this, we need to convert it into a little component so we don't have so much repeated code, but that can wait until then!

File: styles/global.css (hunk 1 above)
Comment (elisa-a-v) [reply styles/global.css]: Indeed! I thought I tried it and it didn't work, it might have been an issue with the hot reload? I changed it now.

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
Done, thank you!

**Pull request merged**